src/Jobs/NewSiteSSH.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Visiosoft\SiteModule\Jobs;
+
+use phpseclib3\Net\SSH2;
+use Illuminate\Bus\Queueable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+
+class NewSiteSSH implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    protected $server;
+    protected $site;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct($site)
+    {
+        $this->site = $site;
+        $this->server = $site->server;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        $serverPassword = $this->server->getPassword();
+
+        $ssh = new SSH2($this->server->getIp(), 22);
+        $ssh->login('pure', $serverPassword);
```

Comment:
there are 6 duplicate of logging server. Can you create loginAndRunCommand function inside server module

Reply:
to centralize